pkg/imagestore/imagestore_test.go

```diff
@@ -76,7 +76,7 @@ var _ = Context("with a data directory configured", func() {
 					),
 				)
 				version["url"] = ts.URL() + "/some.iso"
-				is, err := NewImageStore(mockEditor, dataDir, []map[string]string{version})
+				is, err := NewImageStore(mockEditor, dataDir, true, []map[string]string{version})
```

Comment:
These were working before so I'd rather keep these as the default (`false`) whenever we can.
Even in tests let's make setting the insecure options the exception.

Reply:
Gotcha, changed it back to `false`.